### PR TITLE
Commit funcionalidad calculadora de diluciones

### PR DESCRIPTION
**Pull request by DaniielEF**
## 🔍 Overview

This PR adds a utility to calculate dilution concentrations using the formula:

> C₁ × V₁ = C₂ × V₂

It allows the user to input three values (C₁, V₁, C₂) and computes the fourth, currently only V₂, supporting scientific workflows in lab environments or educational contexts.

---

## 🎯 Purpose

- Automate dilution calculations for lab and educational purposes
- Ensure precise results and reduce manual calculation errors
- Provide a reusable function for future chemistry modules

---

## ✅ Features

- Inputs: C₁ (initial concentration), V₁ (initial volume) and C₂ (final concentration)
- Outputs the value V₂ (final volume) based on the provided three
- Input validation:
  - Only numeric, positive values allowed
  - Throws errors when inputs are insufficient or invalid
- Unit agnostic: Assumes consistent units (all concentrations in Mol, all volumes in mL)

---

## 🧪 Testing

- ✅ Unit tests for:
  - Correct computation of missing values
  - Error handling for invalid inputs
  - Edge cases (e.g., 0 or negative values)
- ✅ Manual testing in browser/dev tool (if frontend)
- ✅ Reviewed against known test cases from standard chemistry problems

---

## 📂 Files Modified

- `Customhooks/Useform.tsx`
-  `components/DissolutionCalculator.tsx`

---

## 📝 Notes

- This function can be reused for serial dilution calculations in future PRs
- In the future, we could support unit conversion (e.g., mL to L)
- pending to implement the history of calculations

---

## 🔗 Related Issues

Closes #1– Add dilution calculation utility
Related to #2– Chemistry tools foundation

### CODE REVIEW

File: src/components/DissolutionCalculator.tsx [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+}
+
+const DissolutionCalculator = () => {
+
+  const { form, history, handleChange, setField, saveHistory } = useForm<FormData>({
+    measureName: 'Muestra ',
+    concentrationPatron: 0,
+    finalConcentration: 0,
+    finalVolume: 1,
+  });
```
Comment (DaniielEF) [inline src/components/DissolutionCalculator.tsx:20]: Note for reviewers: this is for initial values
